theano/tensor/tests/test_opt.py

```diff
@@ -3021,6 +3023,8 @@ def test_incsubtensor_x_zeros(self):
                      if isinstance(n.op, tensor.IncSubtensor)]
         assert(len(inc_nodes) == 1)
         assert(inc_nodes[0].op.set_instead_of_inc is False)
+        test_X = numpy.random.random((4, 4)).astype(theano.config.floatX)
+        assert(numpy.any(f(test_X) == test_X))
```

Comment:
@Faruk-Ahmed  numpy.any? A little slip? 

Anyway it would be better to use utt.assert_allclose than numpy.all.